resources/js/Component/Menu.vue

```diff
@@ -9,7 +9,7 @@
             </b-navbar-item>
         </template>
         <template #start v-if="user">
-            <b-navbar-item href="#">
+            <b-navbar-item href="/albums">
```

Comment:
```suggestion
            <b-navbar-item :href="route('albums.index')">

```
Should use route instead of simple href